tools/submission/submission-checker.py

```diff
@@ -0,0 +1,639 @@
+"""
+A checker for mlperf inference submissions
+"""
+
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import argparse
+import json
+import logging
+import os
+import re
+import sys
+
+# pylint: disable=missing-docstring
+
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger("main")
+
+
+MODEL_CONFIG = {
+    "v0.5": {
+        "models": ["ssd-small", "ssd-large", "mobilenet", "resnet", "gnmt"],
+        "required-scenarios-datacenter": {
+            # anything goes
+        },
+        "optional-scenarios-datacenter": {
+            # anything goes
+        },
+        "required-scenarios-edge": {
+            # anything goes
+        },
+        "optional-scenarios-edge": {
+            # anything goes
+        },
+        "accuracy-target": {
+            "mobilenet": ("acc", 71.68 * 0.98),
+            "resnet": ("acc", 76.46 * 0.99),
+            "ssd-small": ("mAP", 22 * 0.99),
+            "ssd-large": ("mAP", 20 * 0.99),
+            "gnmt": ("bleu", 23.9 * 0.99),
+        },
+        "performance-sample-count": {
+            "mobilenet": 1024,
+            "resnet": 1024,
+            "ssd-small": 256,
+            "ssd-large": 64,
+            "gnmt": 3903900,
+        },
+        "seeds": {
+            "qsl_rng_seed": 3133965575612453542,
+            "sample_index_rng_seed": 665484352860916858,
+            "schedule_rng_seed": 3622009729038561421,
+        },
+    },
+    "v0.7": {
+        "models": [
+            "ssd-small", "ssd-large", "resnet", "rnnt",
+            "bert", "bert-99", "bert-99.9",
+            "dlrm", "dlrm-99", "dlrm-99.9"
+            "3d-unet", "3d-unet-99", "3d-unet-99.9"
+        ],
+        "required-scenarios-datacenter": {
+            "resnet": ["Server", "Offline"],
+            "ssd-large": ["Server", "Offline"],
+            "rnnt": ["Server", "Offline"],
+            "bert": ["Server", "Offline"],
+            "dlrm": ["Server", "Offline"],
+            "3d-unet": ["Offline"],
+        },
+        "optional-scenarios-datacenter": {
+        },
+        "required-scenarios-edge": {
+            "resnet": ["SingleStream", "Offline"],
+            "ssd-small": ["SingleStream", "Offline"],
+            "ssd-large": ["SingleStream", "Offline"],
+            "rnnt": ["SingleStream", "Offline"],
+            "bert": ["SingleStream", "Offline"],
+            "dlrm": ["SingleStream", "Offline"],
```

Comment:
dlrm is not an Edge benchmark.

Reply:
Yes, removed.
https://github.com/mlperf/inference_policies/blob/master/inference_rules.adoc#41-benchmarks